src/client.h

```diff
@@ -255,8 +256,8 @@ class CClient : public QObject
     void SetMuteOutStream ( const bool bDoMute ) { bMuteOutStream = bDoMute; }
 
     void SetRemoteChanGain ( const int iId, const float fGain, const bool bIsMyOwnFader );
-    void OnTimerRemoteChanGain();
-    void StartDelayTimer();
+    void OnTimerRemoteChanGainOrPan();
```

Comment:
It would make sense to have `SetRemoteChanPan` next to `SetRemoteChanGain` and before their shared timer methods.

Reply:
Yes, good point. Done.